### PR TITLE
implement generatetoaddress rpc call

### PR DESCRIPTION
**Pull request by vlad9486**
Hello, I added a method. Please, accept.

### CODE REVIEW

File: src/bitcoincore.rs [hunk 1]
```diff
@@ -163,6 +163,20 @@ impl BitcoinRpcApi for BitcoinCoreClient {
         ))
     }
 
+    fn generate_to_address(
+        &self,
+        number_of_blocks: u32,
+        address: &Address,
+    ) -> Result<Result<Vec<BlockHash>, RpcError>, ClientError> {
```
Comment (thomaseizinger) [inline src/bitcoincore.rs]: :question: Without checking the documentation: The result of `generate_to_address` is probably not a `Vec<BlockHash>`?

Comment (D4nte) [reply src/bitcoincore.rs]: It is https://bitcoincore.org/en/doc/0.16.0/rpc/generating/generatetoaddress/

Comment (thomaseizinger) [reply src/bitcoincore.rs]: Interesting! My bad then. We still need a test for the method though :)

**Comment by thomaseizinger** [comment]
Closed due to inactivity.

**Pull request closed without merging**